src/Simplic.DocumentProcessing.Service/Pdf/PdfService.cs

```diff
@@ -28,5 +28,31 @@ public int GetPageCount(byte[] pdf)
                 }
             }
         }
+
+        /// <summary>
+        /// Adds an empty page.
+        /// </summary>
+        /// <param name="pdf">Pdf</param>
+        /// <param name="pageNumber">Index of the new page, starting with 1 which will insert an empty page as the first page</param>
+        /// <returns>The resulting Pdf-blob</returns>
+        public byte[] AddEmptyPage(byte[] pdf, int pageNumber)
+        {
+            var pdfToTiffService = CommonServiceLocator.ServiceLocator.Current.GetInstance<PdfToTiffService>();
+            using (var stream = new MemoryStream(pdf))
+            {
+                using (var pdfInstance = GdPictureHelper.GetPDFInstance())
+                {
+                    pdfInstance.LoadFromStream(stream);
+                    pdfInstance.SelectPage(1);
+                    float pageWidth = pdfInstance.GetPageWidth();
+                    float pageHeight = pdfInstance.GetPageHeight();
+
+                    pdfInstance.InsertPage(pageWidth, pageHeight, pageNumber);
+                    byte[] blob = pdfToTiffService.Convert(pdf);
+                    pdfInstance?.CloseDocument();
+                    return blob;

```

Comment:
@BettelsPK  here is a major issue, why do you convert the PDF to tiff? The resulting-object should be a pdf.

Take a look here: https://github.com/simplic/simplic-document-processing/blob/master/src/Simplic.DocumentProcessing.Service/PdfLayer/PdfLayerService.cs#L47